test/EndToEnd/NuGet.Client.EndToEnd.TestData.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>NuGet.Client.EndToEnd.TestData</id>
+    <version>1.0.0</version>
+    <title>Test data for NuGet.Client EndToEnd tests</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
```

Comment:
I'd remove this line and allow the toolset to generate it. 

That's our recommendation to our external customers so we should do the same.